libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -5,23 +5,27 @@
       matInput
       type="search"
       placeholder="Search for books to add to your reading list"
+      (input)="searchBooks()"
```

Comment:
This is part of the task 2 (instant search) so you can leave this part out fr this task.

But note that `searchBooks()` will be called multiple times as the user is typing, and since it is async the ordering might be wrong. Something to keep in mind for task 2.